src/platforms/python/common/integrations/aws-lambda/manual-layer/index.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: AWS Lambda Layer
+description: "Learn how to manually set up Sentry with a Layer"
+sidebar_order: 1000
+---
+
+You can add Sentry to your AWS Lambda function by adding the Sentry AWS Lambda Layer to your function.
+
+## Install the layer
+
+Navigate to your Lambda function:
+
+![](lambda-function-detail.png)
+
+Scroll down to the **Layers** section and click **Add a Layer**:
+
+![](lambda-function-detail-layers.png)
+
+**Specify an ARN** tab as illustrated:
+
+![](lambda-function-add-layer.png)
+
+Finally, set the region and copy the provided ARN value into the input.
+
+<LambdaLayerDetail canonical="aws-layer:python" />
+
+## Configure the Sentry SDK
+
+For Sentry to instrument your Lambda function you need to change the handler of your function and set the Sentry DSN using environment variables.
+
+### Set the Sentry Handler
+
+The Sentry layer will wrap your function handler to initialise the Sentry SDK in your function.
+
+In you Lambda function scroll to **Runtime settings** and click **Edit**:
+
+![](lambda-function-runtime-settings.png)
+
+Note your current **Handler** and change it to `sentry_sdk.integrations.init_serverless_sdk.sentry_lambda_handler`:
+
+![](runtime-settings-handler.png)
+
+After saving open the "Configuration" tab of your Lambda function and set `SENTRY_INITIAL_HANDLER` to the handler you noted before:
```

Comment:
```suggestion
After leavving the "Configuration" tab of your Lambda function open and setting `SENTRY_INITIAL_HANDLER` to the handler you noted before:
```
I'm not sure what the instructions are after this. If they're supposed to be visible in the image I unfortunately can't read them b/c they're too small :(